packages/pages/package.json

```diff
@@ -81,7 +81,8 @@
     "ts-morph": "^19.0.0",
     "typescript": "^5.1.6",
     "vite": "^4.4.9",
-    "vite-plugin-node-polyfills": "^0.11.0"
+    "vite-plugin-node-polyfills": "^0.11.0",
+    "yaml": "^2.3.1"
```

Comment:
js-yaml is already imported. Do we need this one too?

Reply:
I found this one easier to work with. It's also being updated more frequently. Last update 1 month ago vs 2 years for js-yaml

Reply:
Ah ok. Can you remove this one then and update my usage of it? Should be easy to swap out.